### PR TITLE
Stale bot for pull requests reloaded

### PR DESCRIPTION
**Pull request by m-kuhn**
## Description

Christmas replacement for stale bot based on github workflows

### CODE REVIEW

File: .github/workflows/stale_pr.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+name: "Close stale issues"
```
Comment (m-kuhn) [inline .github/workflows/stale_pr.yml:1]:
```suggestion
name: "Handle stale pull requests"
```

**Comment by roya0045** [comment]
@m-kuhn was the stalebot supposed to remove the tag upong interaction or does it need to be done manually by someone with the rights? This might be bad for small time contributor like me who can't remove the tags and may get their PR closed.

**Comment by m-kuhn** [comment]
Ideally it does remove the tag as the old one did, does it not?

**Comment by roya0045** [comment]
It might have a delay? For the moment the tag was not removed on #39544 we'll see if it gets removed within 24 hours I guess.

**Pull request merged**